src/attr/converters.py

```diff
@@ -7,6 +7,26 @@
 from ._make import NOTHING, Factory
 
 
+def chain(*converters):
```

Comment:
Please don't define it twice. Just import it from ._make just like we do with validators._and.

Reply:
Done

Reply:
Done